### ISSUE TITLE
GAN Slimming: All-in-One GAN Compression by A Unified Optimization Framework

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#71]
# TL;DR
Research to reduce the size of the GAN generator. A combination of distillation, quantization, and pruning. Quantization is usually non-differentiable, but they use pseudo-gradient to enable E2E training. They have successfully reduced the size of an existing model to 1/47th of its original size.
![Original result](https://user-images.githubusercontent.com/24524018/92557525-5bb0e000-f2a7-11ea-8c36-00c4572e17d4.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2008.11062

# Submission Dates(yyyy/mm/dd)
2020/08/25

# Authors and institutions
Haotao Wang, Shupeng Gui, Haichuan Yang, Ji Liu, Zhangyang Wang
- University of Texas at Austin
- University of Rochester
- AI Platform, Ytech Seattle AI Lab, FeDA Lab, Kwai Inc.

# Methods

# Results

# Comments
ECCV 2020
